etc/barrier.profile

```diff
@@ -0,0 +1,47 @@
+# Firejail profile for barrier
+# Description: Keyboard and mouse sharing application
+# This file is overwritten after every install/update
+# Persistent local customizations
+include barrier.local
+# Persistent global definitions
+include globals.local 
+
+noblacklist ${HOME}/.config/Debauchee/Barrier.conf
+noblacklist ${HOME}/.local/share/barrier
+noblacklist ${PATH}/openssl
```

Comment:
most programs shouldn't need the openssl program, only the library.
the program is commonly used by linux ransomware

Reply:
/usr/bin/openssl appeared as a blacklist violation in syslog. I think the program uses the CLI app to create certs during setup rather than through a library.